scalardb-sample/README.md

```diff
@@ -1,162 +1,124 @@
-      "order_id": "TEXT",
-      "customer_id": "INT",
-      "timestamp": "BIGINT"
-    }
-  },
-  "sample.statements": {
-    "transaction": true,
-    "partition-key": [
-      "order_id"
-    ],
-    "clustering-key": [
-      "item_id"
-    ],
-    "columns": {
-      "order_id": "TEXT",
-      "item_id": "INT",
-      "count": "INT"
-    }
-  },
-  "sample.items": {
-    "transaction": true,
-    "partition-key": [
-      "item_id"
-    ],
-    "columns": {
-      "item_id": "INT",
-      "name": "TEXT",
-      "price": "INT"
-    }
-  }
-}
-```
+The sample e-commerce application shows how users can order and pay for items by using a line of credit.
 
-All the tables are created in the `sample` namespace.
+The database that you will be using in the sample application is Cassandra. Although Cassandra does not provide ACID transaction capabilities, you can make transactions ACID compliant by having your application connect to the database through ScalarDB.
 
-- `sample.customers`: a table that manages customers' information
-    - `credit_limit`: the maximum amount of money a lender will allow each customer to spend when using a credit card
-    - `credit_total`: the amount of money that each customer has already spent by using the credit card
-- `sample.orders`: a table that manages order information
-- `sample.statements`: a table that manages order statement information
-- `sample.items`: a table that manages information of items to be ordered
+{% capture notice--info %}
+**Note**
 
-The Entity Relationship Diagram for the schema is as follows:
+Since the focus of the sample application is to demonstrate using ScalarDB, application-specific error handling, authentication processing, and similar functions are not included in the sample application. For details about exception handling in ScalarDB, see [Handle exceptions](https://github.com/scalar-labs/scalardb/blob/master/docs/api-guide.md#handle-exceptions).
+{% endcapture %}
 
-![ERD](images/ERD.png)
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### What you can do in this sample application
+
+The sample application supports the following types of transactions:
+
+- Get customer information.
+- Place an order by using a line of credit.
+  - Checks if the cost of the order is below the customer's credit limit.
+  - If the check passes, records the order history and updates the amount the customer has spent.
+- Get order information by order ID.
+- Get order information by customer ID.
+- Make a payment.
+  - Reduces the amount the customer has spent.
+
+## Prerequisites
 
-### Transactions
+- One of the following Java Development Kits (JDKs):
+  - [Oracle JDK 8](https://www.oracle.com/java/technologies/downloads/#java8)
+  - [Oracle JDK 11](https://www.oracle.com/java/technologies/downloads/#java11)
+  - [OpenJDK 8](https://openjdk.org/install/)
+- [Docker](https://www.docker.com/get-started/) with [Docker Compose](https://docs.docker.com/compose/install/)
```

Comment:
Do we need to specify which versions of Docker and Docker Compose here?

Reply:
## Docker version

Int the `docker-compose.yaml` for ScalarDB samples, we use `version: "3.5"` file format.
https://github.com/scalar-labs/scalardb-samples/blob/main/scalardb-sample/docker-compose.yml#L1

And, the file format `3.5` is supported by Docker Engine `17.12.0+`.
https://docs.docker.com/compose/compose-file/compose-versioning/#compatibility-matrix

Also, it seems that there is no clear definition of EoL in the Docker project, but of course, old versions have no update.

This is no official information, but, at this time, it seems that `20.10` is the oldest version that has updates.
https://endoflife.date/docker-engine

So, I think it is better to use at least `20.10`.

## Docker Compose version

Docker Compose v1 is already EoL. So, I think it is better to use Docker Compose v2.

https://docs.docker.com/compose/compose-file/

## Conclusion

I think the following version information is better in the current document.
```
Docker 20.10+ with Docker Compose v2+
```

```suggestion
- [Docker 20.10+](https://www.docker.com/get-started/) with [Docker Compose v2+](https://docs.docker.com/compose/install/)
```

Reply:
@kota2and3kan Thank you for the explanation! I've added Docker and Docker Compose versions in the way you described. I'll be sure to do the same for other sample tutorials.

Reply:
Thank you for the update!

> I'll be sure to do the same for other sample tutorials.

Sounds good! Thank you!